app/cmd/root.go

```diff
@@ -107,8 +107,8 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&config.Context, "kubecontext", "", "The name of the kube config context to use")
 }
 
-func installCNI(ctx context.Context, cniReady chan struct{}) {
-	installer := cniserver.NewInstaller()
+func installCNI(ctx context.Context, serviceMeshMode string, cniReady chan struct{}) {
```

Comment:
Why not just use the global `config.Mode`?

Reply:
changed according to your suggestion! :-)

Reply:
Sorry I wasn't clear, I mean directly reading `config.Mode` where needed rather than passing the value